.github/workflows/appmap-analysis.yml

```diff
@@ -24,6 +24,7 @@ jobs:
         uses: EndBug/add-and-commit@v9
         with:
           add: appmap.yml
+          push: origin HEAD:${{ github.head_ref }}
```

Comment:
OK, maybe this is better than modifying the checkout?

Reply:
Never mind :-) It's not working https://github.com/land-of-apps/rails-7-app/actions/runs/6433013707/job/17469149885?pr=1#step:6:63